image-download

```diff
@@ -85,6 +85,17 @@ def find(name, stores, latest, quiet):
     for store in stores:
         url = urllib.parse.urlparse(urllib.parse.urljoin(store, name))
 
+        # First, try the easy case: for OpenShift proxied stores which send
+        # their own SSL cert initially host name has to match that
```

Comment:
This is a bit confusing -- the OpenShift hosted image server has exactly the same cockpit cert than our e2e and aws ones; we use TLS passthrough, as unfortunately TLS endpoint termination doesn't work for large files (it's a pity, as otherwise we'd have a real certificate on these). It's just that its DNS name is predictable and fixed, so we don't have to use the `--resolve` trick.